linux-cachyos/PKGBUILD

```diff
@@ -640,7 +640,11 @@ prepare() {
 
 build() {
     cd ${srcdir}/${_srcname}
-    make ${BUILD_FLAGS[*]} -j$(nproc) all
+    if [ -z "${_use_llvm_lto}" ] && ([ -z "${CC}" ] || [ "${CC}" != "clang" ] || [ "${CC}" == "gcc" ]); then
```

Comment:
Why do you need all these CC checks? It's enough to check `_use_llvm_lto` to know if we are using Clang or not. You should also use _use_llvm_lto `[ "$_use_llvm_lto" = "none" ]` instead of checking for empty.

Reply:
> My concern for that is to not overwrite user decision. What if he don't want lto, but want to compile with Clang, and has CC=clang in his makepkg?
> 

Then nothing happens. Environment variables like CC and CCX are ignored when building the kernel. You can check this for yourself.  The user should use the only way to enable Clang is through the kernel config.